packages/terra-form-input/tests/jest/Input.test.jsx

```diff
@@ -16,6 +16,12 @@ describe('Input', () => {
     expect(wrapper).toMatchSnapshot();
   });
 
+  it('should render with a type of password and pattern prop value', () => {
+    const input = <Input type="password" pattern=".{6,}" />;
+    const wrapper = render(input);
```

Comment:
Is there a specific reason to do a full render vs a shallow render here?

Reply:
I updated that test (and the others) to run shallow, and the snaps all seem to look good. Look them over and let me know!